interp/notation_ops.ml

```diff
@@ -441,7 +445,8 @@ let notation_constr_and_vars_of_glob_constr recvars a =
         aux' c
   and aux' x = DAst.with_val (function
   | GVar id -> if not (Id.equal id ldots_var) then add_id found id; NVar id
-  | GApp (g,args) -> NApp (aux g, List.map aux args)
+  | GApp (g,[]) -> NApp (aux g,[]) (* Encoding @foo *)
+  | GApp (g,args) -> let a,args = List.sep_last args in mkNApp1 (aux (DAst.make (GApp (g, args))), aux a)
```

Comment:
Maybe a short comment for this case too? To make explicit what it corresponds to.

Reply:
I don't know. I'd be happy to be able to answer your concern for documentation but there is also a compromise to find. Somehow it seems to me that the `sep_last` is self-explanatory and faster to read than a comment. Moreover, adding a comment would break the conciseness of this group of lines and make the code less easily graspable as a whole. Also, if we start to make a comment here, why not to add one at every line of the match? For instance there is more specificities in the `add_found` lines than in the `App` line which is after all rather canonical.

In any case, feel free to push whatever comment you'd like to see though.